packages/components/pie-switch/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@justeattakeaway/pie-toggle-switch",
-  "description": "PIE Design System Toggle Switch built using Web Components",
-  "version": "0.15.0",
+  "name": "@justeattakeaway/pie-switch",
+  "description": "PIE Design System Switch built using Web Components",
+  "version": "0.0.0",
```

Comment:
I'm not sure what this will do to the component changelog – it might look a bit strange though.

As it's only a prerelease version, I'd likely leave this to be v0.15, as then the changelog will be correct in terms of describing the changes.